hoomd/mpcd/pytest/test_collide.py

```diff
@@ -226,9 +226,12 @@ def test_rigid_collide(
             initial_snap.particles.angmom[:] = [angmom_rigid]
 
             # place the mpcd particles on top of constituents
```

Comment:
Please document this code change in this comment so it is clear why it is needed
```suggestion
            # place the mpcd particles on top of constituents, accounting for PBCs
```